docs/resources/rest_managed.md

```diff
@@ -115,9 +119,29 @@ When children need to be imported they must be specified by appending them to th
 terraform import aci_rest_managed.example_tenant uni/tn-{name}:{child-rn-1},{child-rn-N}
 ```
 
+!> The use of the colon-separated format to import children for the resource is deprecated and will be removed in the next release. Please use the JSON format string shown below to import children, instead of using a colon-separated import statement.
```

Comment:
```suggestion
!> The use of the previously supported colon-separated format to import children for the resource is deprecated and will be removed in the next release as it creates issue with DN containing colon. Please use the JSON format string shown above to import children.
```

Move this block after the next import command.

Reply:
Done!